packages/babel-plugin/src/processJSXHTMLTag.ts

```diff
@@ -29,43 +29,44 @@ export const processJSXHTMLTag = (path: NodePath<t.JSXOpeningElement>) => {
     };
 
     // Add the style rule to the sheet and get the generated class name.
-    const className = sheet.addRule(style);
+    const generatedClassName = sheet.addRule(style);
 
-    const classNameAttrs: t.Expression[] = [t.stringLiteral(className)];
+    const generatedClassNameAttr = t.stringLiteral(generatedClassName);
+    const classNameAttrs: t.Expression[] = [generatedClassNameAttr];
 
-    for (const attr of filteredAttributes) {
-      if (
+    const existsClassNameAttr = path.node.attributes.find(
+      (attr): attr is t.JSXAttribute =>
         t.isJSXAttribute(attr) &&
-        t.isJSXIdentifier(attr.name) &&
-        attr.name.name === "className"
+        t.isJSXIdentifier(attr.name, { name: "className" })
+    );
+    if (existsClassNameAttr) {
+      if (t.isStringLiteral(existsClassNameAttr.value)) {
+        classNameAttrs.push(existsClassNameAttr.value);
+      }
+      if (
+        t.isJSXExpressionContainer(existsClassNameAttr.value) &&
+        t.isExpression(existsClassNameAttr.value.expression)
       ) {
-        if (t.isStringLiteral(attr.value)) {
-          classNameAttrs.push(attr.value);
-        } else if (
-          t.isJSXExpressionContainer(attr.value) &&
-          t.isExpression(attr.value.expression)
-        ) {
-          classNameAttrs.push(attr.value.expression);
-        }
-        return attr.value;
```

Comment:
This code is the cause of the Issue: whenever a className is specified in the JSX, it is returned here. Therefore, subsequent processing will not be executed.

Reply:
I see 👍